File: CS/SpreadsheetChartAPISamples/CodeExamples/Protection.cs
using DevExpress.Spreadsheet;
using DevExpress.Spreadsheet.Charts;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SpreadsheetChartAPIActions
{
    public static class Protection
    {
        static void ProtectChart(IWorkbook workbook)
        {
            #region #ProtectChart
            Worksheet worksheet = workbook.Worksheets["chartTask3"];
            workbook.Worksheets.ActiveWorksheet = worksheet;

            // Create a chart and specify its location.
            Chart chart = worksheet.Charts.Add(ChartType.ColumnClustered, worksheet["B2:D4"]);
            chart.TopLeftCell = worksheet.Cells["H2"];
            chart.BottomRightCell = worksheet.Cells["N14"];

            // Specify the chart style.
            chart.Style = ChartStyle.ColorDark;

            // Apply the chart protection.
            chart.Options.Protection = ChartProtection.All;

            #endregion #ProtectChart
        }

    }
}

File: VB/SpreadsheetChartAPISamples/CodeExamples/Protection.vb
Imports DevExpress.Spreadsheet
Imports DevExpress.Spreadsheet.Charts
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace SpreadsheetChartAPIActions

    Public Module Protection

        Private Sub ProtectChart(ByVal workbook As DevExpress.Spreadsheet.IWorkbook)
'#Region "#ProtectChart"
            Dim worksheet As DevExpress.Spreadsheet.Worksheet = workbook.Worksheets("chartTask3")
            workbook.Worksheets.ActiveWorksheet = worksheet
            ' Create a chart and specify its location.
            Dim chart As DevExpress.Spreadsheet.Charts.Chart = worksheet.Charts.Add(DevExpress.Spreadsheet.Charts.ChartType.ColumnClustered, worksheet("B2:D4"))
            chart.TopLeftCell = worksheet.Cells("H2")
            chart.BottomRightCell = worksheet.Cells("N14")
            ' Specify the chart style.
            chart.Style = DevExpress.Spreadsheet.Charts.ChartStyle.ColorDark
            ' Apply the chart protection.
            chart.Options.Protection = DevExpress.Spreadsheet.Charts.ChartProtection.All
'#End Region  ' #ProtectChart
        End Sub
    End Module
End Namespace
